src/components/IconLayer.tsx

```diff
@@ -14,13 +15,13 @@ type IconLayerProps = {
 const IconLayer: React.FC<IconLayerProps> = (props) => {
   const iconProps = { style: props.style };
   switch(props.type) {
-    case "fill-extrusion": return <BsDiamondFill {...iconProps} />;
-    case "raster": return <BsDiamond {...iconProps} />;
+    case "fill-extrusion": return <MdOutlineVerticalShades {...iconProps} />;
+    case "raster": return <BsDiamondFill {...iconProps} />;
```

Comment:
raster should likely not be the same as fill.

How about one of these:
<img width="108" height="89" alt="image" src="https://github.com/user-attachments/assets/2fd97a56-43bd-43a0-9429-eb8b45719b07" /><img width="113" height="103" alt="image" src="https://github.com/user-attachments/assets/ab125e15-36a2-4b5a-bf8c-8e6d0375efda" /><img width="109" height="102" alt="image" src="https://github.com/user-attachments/assets/2755ce06-f2eb-4160-b715-2b9127e142eb" />